meetings/2022-01-27.md

```diff
@@ -0,0 +1,151 @@
+# W3C Solid Community Group: Notifications Panel
+
+* Date: 2022-01-27T15:00:00Z
+* Call: https://meet.jit.si/solid-notifications
+* Chat: https://gitter.im/solid/notifications-panel
+* Repository: https://github.com/solid/notifications-panel
+* Status: Published
+
+## Present
+* [Sarven Capadisli](https://csarven.ca/#i)
+* Rahul Gupta
+* Aaron Coburn
+* Pete Edwards
+* Jamie Fiedler
+* elf Pavlik
+
+---
+
+## Announcements
+
+### Meeting Recordings and Transcripts
+* No audio or video recording, or automated transcripts without consent. Meetings are transcribed and made public. If consent is withheld by anyone, recording/retention must not occur.
+* Join queue to talk.
+
+
+### Participation and Code of Conduct
+* [Join the W3C Solid Community Group](https://www.w3.org/community/solid/join), [W3C Account Request](http://www.w3.org/accounts/request), [W3C Community Contributor License Agreement](https://www.w3.org/community/about/agreements/cla/)
+* [Solid Code of Conduct](https://github.com/solid/process/blob/main/code-of-conduct.md), [Positive Work Environment at W3C: Code of Ethics and Professional Conduct](https://www.w3.org/Consortium/cepc/)
+* Operating principle for effective participation is to allow access across disabilities, across country borders, and across time. Feedback on tooling and meeting timing is welcome.
+* If this is your first time, welcome! please introduce yourself.
+
+
+### Scribes
+* Aaron Coburn
+
+---
+
+## Topics
+
+* SC: Topics related to ~FPWD excluded from this meeting agenda because there are dependencies (as listed here). They will be picked up once these dependencies are resolved.
+
+### Server accepts GET, HEAD and OPTIONS requests targeting subscription resource.
+URL: https://github.com/solid/notifications/pull/29#issuecomment-1009840586
+
+* SC: We agreed to PR this but didn't get to it. I can take this ACTION.
+* SC: Solid Notifications will rely on the Solid Protocol, and so for any resource GET/HEAD/OPTIONS should be supported. Because this is a non-container resource (to date) and we expect it to receive a POST request, we may need to say something in particular about this.
+* eP: In PR #33 the terminology may be relevant here. For example, a "subscription container".
+* SC: the use of subscription container was mentioned in the comments of that PR. We should address that in the PR. If we open up the container possibility, we will need to consider the access control angle
```

Comment:
```suggestion
* SC: the use of subscription container was originally mentioned in the comment of https://github.com/solid/notifications/pull/29#issuecomment-1009840586 .
* SC: Is that what you're referring to?
* eP: Yes.
* SC: We should address that in the PR. If we open up the container possibility, we will need to consider the access control angle
```

Reply:
Suggested this for awareness of the change but forgot to commit before merging the PR. Pushed in https://github.com/solid/notifications-panel/commit/8bcbe6a47e7e2487ac116f8f6df6651b19dbbb9f